WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingViewModel.swift

```diff
@@ -143,6 +143,14 @@ final class CardPresentPaymentsOnboardingViewModel: ObservableObject, PaymentSet
     var didUpdate: (() -> Void)? = nil
 }
 
+extension CardPresentPaymentsOnboardingViewModel: Equatable {
+    static func == (lhs: CardPresentPaymentsOnboardingViewModel, rhs: CardPresentPaymentsOnboardingViewModel) -> Bool {
+        lhs.state == rhs.state &&
+        lhs.userIsAdministrator == rhs.userIsAdministrator &&
+        lhs.learnMoreURL == rhs.learnMoreURL
```

Comment:
```suggestion
        lhs.learnMoreURL == rhs.learnMoreURL &&
        lhs.shouldShow == rhs.shouldShow
```

Reply:
Great catch, I didn't realize there are variables below the functions 😅 was it intentional to organize the functions & variables this way in payments at least?

Updated in https://github.com/woocommerce/woocommerce-ios/pull/14224/commits/84455c4377b0ac4fc47d72adbdfc723bfabc9418.